templates/main/index.html.twig

```diff
@@ -4,7 +4,7 @@
     ordre: { title: "Ordre", desc: "Opret ordrepakker ud fra tasks i Jira", icon: "fa-box-open", path: "/", btnTxt: "Pak en Ordre" },
     sprintplan: { title: "Sprintplan", desc: "Planlægningsoversigt og sprintrapporter baseret på tasks i Jira", icon: "fa-braille", path: "/", btnTxt: "Se Sprintplan" },
     nytProjekt: { title: "Nyt projekt", desc: "Opret et nyt Jira-projekt ud fra en skabelon", icon: "fa-project-diagram", path: "/create-project/new", btnTxt: "Opret Nyt projekt" },
-    udgift: { title: "Udgift", desc: "Opret udgifter i forbindelse med projekter og tasks", icon: "fa-credit-card", path: "/", btnTxt: "Tilføj en Udgift" },
+    udgift: { title: "Udgift", desc: "Opret udgifter i forbindelse med projekter og tasks", icon: "fa-credit-card", path: "/expense/new", btnTxt: "Tilføj en Udgift" },
```

Comment:
Add to Controller instead that will be added by https://github.com/aakb/jira_economics/pull/14